lib/scheduler.pm

```diff
@@ -43,6 +44,8 @@ sub parse_vars {
         $value =~ s/%(.*?)%/get_var($1)/eg;
         $vars{$var} = $value;
     }
+    my $varlog = Dumper(\%vars);
+    diag("parse_vars (VARS parsed from YAML schedule): " . $varlog);
```

Comment:
neatpicking: they are called Settings in openQA, instead of VARS, and we call it 'vars' in our yaml schedule, so something like this would be more clear: `openQA settings parsed from 'vars' section in YAML schedule`. wdyt?
Could we apply the same in [line 132](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12933/files#diff-bbd0c458d484c80e3bf564bd65378809c63c2268756729a9f9319a034783b1abR132) ?

Reply:
ok, documentation also use 'variables' so for that is called vars.json that generated file, so, we could use `Variables parsed from 'vars' section in YAML schedule` or `Variables parsed from YAML schedule`, but it is just a suggestion.